src/test/testFeeds.ts

```diff
@@ -116,6 +70,13 @@ export const getPrefabFeed = (): FeedItem<ActivityContentNote>[] => {
   const address1 = getPrefabSocialAddress(1);
   const address2 = getPrefabSocialAddress(2);
   const address3 = getPrefabSocialAddress(3);
+
+  const imgAttachment = generateImageAttachment(
+    "https://64.media.tumblr.com/bd8d2127a91f57463c2e753cf837ab6e/014df86f4004efef-ec/s1280x1920/adda1023806b71606f83f484a64daa03bce12c8d.jpg"
+  );
+  const note = generateNote("Everyone leave me alone");
+  note.attachment = [imgAttachment];
```

Comment:
[`generateNote()`](https://github.com/LibertyDSNP/sdk-ts/blob/main/src/generators/activityContentGenerators.ts#L24) has an optional `hasAttachment` argument which will automatically add a fake attachment for you.

Reply:
oh! I missed that completely, ok.